spring-cloud-dataflow-server-kubernetes-docs/pom.xml

```diff
@@ -58,14 +58,24 @@
 						<artifactId>asciidoctor-maven-plugin</artifactId>
 						<version>1.5.3</version>
 						<dependencies>
-						  <dependency>
-						    <!-- Needed to resolve URI includes -->
-						    <!-- See https://github.com/asciidoctor/asciidoctor.js/issues/153 -->
-						    <groupId>org.jruby</groupId>
-						    <artifactId>jruby-complete</artifactId>
-						    <version>9.0.0.0</version>
-						  </dependency>
+							<dependency>
```

Comment:
Actually not needed now that we dropped asciidoctor-pdf direct rendering. I will remove on merge